tools/build-wasm-example/src/main.rs

```diff
@@ -73,15 +77,23 @@ fn main() {
             parameters.push("--features");
             parameters.push(&features_string);
         }
+
+        let profile = if cli.debug {
+            "debug"
+        } else {
+            parameters.push("--release");
+            "release"
+        };
+
         let cmd = cmd!(
             sh,
-            "cargo build {parameters...} --profile release --target wasm32-unknown-unknown --example {example}"
+            "cargo build {parameters...} --target wasm32-unknown-unknown --example {example}"
         );
         cmd.run().expect("Error building example");
 
         cmd!(
             sh,
-            "wasm-bindgen --out-dir examples/wasm/target --out-name wasm_example --target web target/wasm32-unknown-unknown/release/examples/{example}.wasm"
+            "wasm-bindgen --out-dir examples/wasm/target/{profile} --out-name wasm_example --target web target/wasm32-unknown-unknown/{profile}/examples/{example}.wasm"
```

Comment:
```suggestion
            "wasm-bindgen --out-dir examples/wasm/target --out-name wasm_example --target web target/wasm32-unknown-unknown/{profile}/examples/{example}.wasm"
```

The output directory should remain the same. The `index.html` expects bindings there.